WordPress/src/main/res/values/strings.xml

```diff
@@ -1023,6 +1023,9 @@
     <string name="error_fetch_remote_site_settings">Couldn\'t retrieve site info</string>
     <string name="error_post_remote_site_settings">Couldn\'t save site info</string>
     <string name="error_open_list_from_notification">This post or page was published on another site</string>
+    <string name="error_fetch_people_list">Couldn\'t retrieve users for the site</string>
+    <string name="error_update_role">Couldn\'t update role of the user</string>
+    <string name="error_remove_user">Couldn\'t remove user</string>
```

Comment:
I've added these strings temporarily and as you can see from the checklist in the master issue: https://github.com/wordpress-mobile/WordPress-Android/issues/3901#issuecomment-217828361 it's on my list to get feedback on them from designers on the main PR that'll be merged to `develop`. Thanks!